README.md

```diff
@@ -114,12 +114,15 @@ Available block directives include:
 - `@public` marks a function public. In most plugins, functions are private by
   default, though this default may be overridden on a per-plugin basis.
 - `@private` marks a function private.
-- `@section name[, id]` allows you to write a new section for the helpfile. The
-  id will be a lowercased version of name if omitted.
+- `@section name[, id]` allows you to write a new section for the
```

Comment:
Nit: Looks like there's a spurious edit wrapping this line early even though it's otherwise identical.